server/game/cards/07-GR/DeEscalation.js

```diff
@@ -0,0 +1,34 @@
+const Card = require('../../Card.js');
+
+class DeEscalation extends Card {
+    // Play: Destroy each creature. Your opponent archives the top 3
+    // cards of their deck.
+    setupCardAbilities(ability) {
+        this.play({
+            effect: 'destroy all creatures{1}',
+            effectArgs: (context) => [
+                context.player.opponent
+                    ? ' and have ' +
+                      context.player.opponent.name +
+                      ' archive the top 3 cards of their deck'
+                    : ''
+            ],
+            gameAction: ability.actions.sequential([
+                ability.actions.destroy((context) => ({ target: context.game.creaturesInPlay })),
+                ability.actions.conditional((context) => ({
```

Comment:
I do not think you need to add conditional here.

```javascript
gameAction: ability.actions.sequential([
                ability.actions.destroy((context) => ({ target: context.game.creaturesInPlay })),
                ability.actions.archive({
                   target: context.player.opponent ? context.player.opponent.deck.slice(0, 3) : []
                })
])
```

Reply:
Done, thanks.